articles/api/management/v2/user-search.md

```diff
@@ -3,44 +3,49 @@ description: This page lists several examples of user search queries using query
 
 ![Select Lucene Syntax](/media/articles/api/user-search-lucene.png)
 
-## Using the Management API
+## Search for Users Using the Management API
+
+You can also search for users using the [Management API](/api/v2). Two of the easiest ways to do this is by either making use of the **API Explorer** or by using **Postman**. These 2 techniques are discussed briefly below, but please note that the Auth0 Management API is a REST API, so you can make API calls using anything that can make HTTP requests, or by using one of the [Auth0 SDKs](/support/matrix#sdks).
 
-### API Explorer
+In order to make requests to the Management API, you will need a token. Please refer to [The Auth0 Management APIv2 Token](/api/management/v2/tokens) for more information.
 
-To search users using the [Management APIv2](/api/v2#!/users/get_users) explorer go to the **Users** section and then select **List or search users**. Click on the scopes for your search. Scroll down to the `q` parameter, this field uses [query string syntax](/api/management/v2/query-string-syntax).
+### Search using the API Explorer
 
-![q parameter](/media/articles/api/search-users-api.png)
+To search users using the [Management API Explorer](/api/v2#!/users/get_users), go to the **Users** section and then select **List or search users**. Scroll down to the `q` parameter. You can use any query string which uses the [query string syntax](/api/management/v2/query-string-syntax) in this field.
 
-In this field enter your query string to search users. [See below](#example-queries) for examples of queries you can run from this field.
+![Searching users in API Explorer](/media/articles/api/search-users-api.png)
 
-### Postman
+### Search using Postman
 
-To search users in [Postman](https://auth0.com/docs/api/postman). First make sure your [enviroment is configured](https://auth0.com/docs/api/postman#configuring-the-postman-environment). Then select the **Management API** as the enviroment you want to work in. For searching users, the API method is `GET ${account.namespace}/api/v2/users` which is under **Users** -> **List or search users**.
+You can also search users using the Postman Collection for the Management API. Make sure you read [Using the Auth0 API with our Postman Collections](https://auth0.com/docs/api/postman) for more information on how to install the collection and also configure your Postman environment correctly.
 
-![Select GET users](/media/articles/api/postman/get-users-postman.png)
+Once you have downloaded the collection, and configured your environment, select the **Management API** collection. Navigate to the **Users** folder and select **List or search users**. You can enter your query in the `q` parameter of the URL:
 
-Then you can run your queries in the **Body** section and click **Send**.
+::: zoomable
```

Comment:
You have to convert the image to .svg format if you want to use the `zoomable` option, as is it does not offer much. You just see a bigger blurred out image. I would simply leave it as png though and remove the zoomable option, we use it mainly for complex flows, not screenshots.